File: arduino/InteractiveLights/SearchUtils.cpp
#include "SearchUtils.h"

int search_utils::indexOf(const char *bytes, int length, int byteValueToFind)
{
    for (auto i = 0; i < length; i++)
    {
        if ((int)bytes[i] == byteValueToFind)
        {
            return i;
        }
    }
    return -1;
}

File: app/Adruino/UserCode/PixelProtocolDisplayDriver/search_utils.ts
export class search_utils {

    public static indexOf(bytes: number[], length: number, byteValueToFind: number) {
        for (let i = 0; i < length; i++) {
            if (bytes[i] == byteValueToFind) {
                return i;
            }
        }
        return -1;
    }

}